search.py

```diff
@@ -79,41 +91,53 @@ def cosine_score(tokens_arr):
         # We will calculate its weight in Step 3
         # This is nicely reflected in the term's PostingList
         # Only documents with Postings of this term will have non-zero score contributions
-        posting_list = find_term(term)
+        posting_list = []
```

Comment:
Previously, i did this kind of thing in a loop but it didnt clear. Had to use "posting_list.clear()" instead. Not sure if this problem would arise here

Reply:
Sure ill look into it. currently debugging an issue with boolean search

Reply:
There seems to be some weird issue with the indexing, because phrasal queries like `fertility treatment` example given by jin dosent seem to register correct positions. still investigating